CMakeLists.txt

```diff
@@ -273,6 +273,9 @@ function(OSQP_BUILD_STATIC_LIB)
     # Transitive dependency on OBJECT library does not work. See https://gitlab.kitware.com/cmake/cmake/-/issues/18682
     target_sources(osqpstatic PRIVATE $<TARGET_OBJECTS:qdldlobject>)
   endif()
+  target_include_directories(
+    osqpstatic PUBLIC "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
+                      "$<INSTALL_INTERFACE:$<INSTALL_PREFIX>/${CMAKE_INSTALL_INCLUDEDIR}/osqp>")
```

Comment:
I don't think this variable will be properly populated until line 333 because this variable is included inside the `GNUInstallDirs` CMake file that we include there. We probably need to move that include to the beginning of the file so that we can reference the path properly.

Reply:
Thanks @imciner2 - Fixed now.